third_party/llvm-project/Path.cpp

```diff
@@ -51,8 +51,7 @@ namespace {
   }
 
   inline char preferred_separator(Style style) {
-    if (real_style(style) == Style::windows)
-      return '\\';
+    // XXX BINARYEN: always use unix-style separators for consistency
```

Comment:
Is this safe?  Seems like it fixes the test failure but will it cause any issues for windows users?

Reply:
It just affects the output of dwarfdump I believe, which is for developer debugging. At least the test suite passes.

But you're right, maybe that's not good enough. Scanning the code, it's hard to be sure there isn't some way that this could affect things, that we don't test atm.